symfony/framework-bundle/3.3/Makefile

```diff
@@ -7,8 +7,13 @@ cache-warmup: cache-clear
 .PHONY: cache-warmup
 
 serve:
-	@echo -e "\033[32;49mServer listening on http://127.0.0.1:8000\033[39m"
-	@echo "Quit the server with CTRL-C."
-	@echo -e "Run \033[32mcomposer require symfony/web-server-bundle\033[39m for a better web server"
-	php -S 127.0.0.1:8000 -t web
+	@if [ -a bin/console ] && [[ `bin/console list server | grep server:start` ]]; then \
+		bin/console server:start; \
+		echo -e "Quit the server with \033[32;49mbin/console server:stop.\033[39m"; \
```

Comment:
ah, it is only explained in the command help indeed, not when running the command